app/scripts/lib/metaRPCClientFactory.js

```diff
@@ -10,6 +11,21 @@ class MetaRPCClient {
     this.requests = new Map();
     this.connectionStream.on('data', this.handleResponse.bind(this));
     this.connectionStream.on('end', this.close.bind(this));
+    this.responseHandled = {};
+  }
+
+  send(id, payload, cb) {
+    this.requests.set(id, cb);
+    this.connectionStream.write(payload);
+    this.responseHandled[id] = false;
+    setTimeout(() => {
+      if (!this.responseHandled[id] && cb) {
+        return cb(new Error('No response from RPC'), null);
```

Comment:
This is interesting. You're inside of an arrow function, which will get executed asynchronously, after the `send` function completes. Judging by the comment below, maybe you need to give the `send` method a return type of `void`? Then maybe you wouldn't get a linter error.

Reply:
The error is actually within the setTimeout function

`Expected to return a value at the end of arrow function`

Reply:
Ah, interesting. No worries then.